### ISSUE TITLE
Remove Keyword filter from cardFilter.js

### ISSUE THREAD

**Issue by datanoh** [trekcc-ops/gemp-stccg#6]
Keyword filtering has been removed from the server side of the code, but is still present in cardFilter.js, as well as cardFilter.test.js and some commented-out lines in deckBuild.html.

Eventually keyword will be re-added, but not until 2E is implemented.

**Comment by datanoh** [comment]
Removing this because there's another issue that covers this
